src/main/java/com/SpringBootProject/DaoServices/ClientsDaoInterface.java

```diff
@@ -0,0 +1,41 @@
+package com.SpringBootProject.DaoServices;
+
+import java.util.List;
+
+import com.SpringBootProject.Entity.Clients;
+
+public interface ClientsDaoInterface {
+	/**
+	 * This method select all client form db
+	 * @return List<Clients>
+	 */
+	List<Clients> GetAllClients();
```

Comment:
1. The first symbol of methods should begin with a lowercase symbol.
2. You should limit the selecting of rows from the database as it can contain millions of rows.
3. Take word "...Clients" from name of the method as you write it in an interface that will be working with clients.